containers/Dockerfile.alpine

```diff
@@ -1,8 +1,8 @@
 FROM node:8-alpine
 
-RUN apk add --no-cache --update bash \
-        && addgroup -g 65533 teleirc \
-        && adduser -s /bin/bash -h /opt/teleirc -D -H teleirc -u 65533 -G teleirc
+RUN apk add --no-cache --update bash build-base python \
+&& addgroup -g 65532 teleirc \
+&& adduser -s /bin/bash -h /opt/teleirc -D -H teleirc -u 65532 -G teleirc
```

Comment:
@robbyoconnor Just curious, is there any significance behind changing the group and user IDs?

Reply:
Yup -- I was transparent about that and I just kept things uniform with the fedora image